gui/gui/src/TRootCanvas.cxx

```diff
@@ -1473,6 +1473,35 @@ void TRootCanvas::ShowEditor(Bool_t show)
    UInt_t h = GetHeight();
    UInt_t s = fHorizontal1->GetHeight();
 
+   auto lambda_show = [&, this](Bool_t display) {
```

Comment:
The parameter is always `== show`, right? So why pass it as a parameter at all? Why not use it from the lambda?

Reply:
Oh yes, you're right...